ERROR - 2024-03-02 17:52:50 --> Error connecting to the database: mysqli_sql_exception: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı.
 in C:\xampp\htdocs\notlarimnet\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\notlarimnet\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'notlarimnet', 3306, '', 0)
#1 C:\xampp\htdocs\notlarimnet\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\notlarimnet\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\notlarimnet\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\notlarimnet\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\notlarimnet\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\notlarimnet\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\notlarimnet\app\Controllers\ProcessController.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
#9 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
#10 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\notlarimnet\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\notlarimnet\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı.
 in C:\xampp\htdocs\notlarimnet\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\notlarimnet\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\notlarimnet\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\notlarimnet\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\notlarimnet\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\notlarimnet\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\notlarimnet\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\notlarimnet\app\Controllers\ProcessController.php(30): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
#8 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
#9 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\notlarimnet\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\notlarimnet\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2024-03-02 17:52:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı.

in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\ProcessController.php(30): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 18:11:46 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "home.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home.php')
 2 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('home', [], true)
 3 APPPATH\Controllers\ProcessController.php(62): view('home')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->page_home()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 18:12:04 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "home.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home.php')
 2 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('home', [], true)
 3 APPPATH\Controllers\ProcessController.php(62): view('home')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->page_home()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 18:14:35 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "home.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home.php')
 2 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('home', [], true)
 3 APPPATH\Controllers\ProcessController.php(62): view('home')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->page_home()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 18:27:23 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "home.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home.php')
 2 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('home', [], true)
 3 APPPATH\Controllers\ProcessController.php(62): view('home')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->page_home()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 18:29:35 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "ali.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('ali.php')
 2 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('ali', [], true)
 3 APPPATH\Controllers\ProcessController.php(62): view('ali')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->ali()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 18:30:45 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "ali.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('ali.php')
 2 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('ali', [], true)
 3 APPPATH\Controllers\ProcessController.php(55): view('ali')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->login()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 18:32:08 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "ali.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('ali.php')
 2 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('ali', [], true)
 3 APPPATH\Controllers\ProcessController.php(57): view('ali')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->ali()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
ERROR - 2024-03-02 21:01:59 --> mysqli_sql_exception: Duplicate entry 'deneme1' for key 'user_username' in C:\xampp\htdocs\notlarimnet\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\notlarimnet\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\notlarimnet\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\notlarimnet\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\notlarimnet\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\notlarimnet\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\notlarimnet\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\notlarimnet\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\notlarimnet\app\Controllers\ProcessController.php(34): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
#9 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
#10 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\notlarimnet\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\notlarimnet\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-03-02 21:01:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'deneme1' for key 'user_username'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\ProcessController.php(34): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:01:59 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'deneme1' for key 'user_username'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`username`, `password`) VALUES (\'deneme1\', \'123\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`username`, `password`) VALUES (\'deneme1\', \'123\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\ProcessController.php(34): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:01:59 --> [Caused by] mysqli_sql_exception: Duplicate entry 'deneme1' for key 'user_username'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `user` (`username`, `password`) VALUES (\'deneme1\', \'123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`username`, `password`) VALUES (\'deneme1\', \'123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`username`, `password`) VALUES (\'deneme1\', \'123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ProcessController.php(34): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
ERROR - 2024-03-02 21:06:56 --> mysqli_sql_exception: Duplicate entry 'asd' for key 'user_username' in C:\xampp\htdocs\notlarimnet\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\notlarimnet\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\notlarimnet\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\notlarimnet\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\notlarimnet\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\notlarimnet\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\notlarimnet\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\notlarimnet\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\notlarimnet\app\Controllers\ProcessController.php(34): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
#9 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
#10 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\notlarimnet\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\notlarimnet\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-03-02 21:06:56 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'asd' for key 'user_username'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\ProcessController.php(34): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:06:56 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'asd' for key 'user_username'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\ProcessController.php(34): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:06:56 --> [Caused by] mysqli_sql_exception: Duplicate entry 'asd' for key 'user_username'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ProcessController.php(34): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
ERROR - 2024-03-02 21:08:02 --> mysqli_sql_exception: Duplicate entry 'asd' for key 'user_username' in C:\xampp\htdocs\notlarimnet\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\notlarimnet\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\notlarimnet\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\notlarimnet\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\notlarimnet\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\notlarimnet\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\notlarimnet\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\notlarimnet\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\notlarimnet\app\Controllers\ProcessController.php(34): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
#9 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
#10 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\notlarimnet\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\notlarimnet\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-03-02 21:08:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'asd' for key 'user_username'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\ProcessController.php(34): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:08:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'asd' for key 'user_username'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\ProcessController.php(34): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:08:02 --> [Caused by] mysqli_sql_exception: Duplicate entry 'asd' for key 'user_username'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ProcessController.php(34): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
ERROR - 2024-03-02 21:16:59 --> mysqli_sql_exception: Duplicate entry 'asd' for key 'user_username' in C:\xampp\htdocs\notlarimnet\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\notlarimnet\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\notlarimnet\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\notlarimnet\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\notlarimnet\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\notlarimnet\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\notlarimnet\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\notlarimnet\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\notlarimnet\app\Controllers\ProcessController.php(34): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
#9 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
#10 C:\xampp\htdocs\notlarimnet\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\notlarimnet\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\notlarimnet\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-03-02 21:16:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'asd' for key 'user_username'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\ProcessController.php(34): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:16:59 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'asd' for key 'user_username'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\ProcessController.php(34): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:16:59 --> [Caused by] mysqli_sql_exception: Duplicate entry 'asd' for key 'user_username'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`username`, `password`) VALUES (\'asd\', \'s\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ProcessController.php(34): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:30:11 --> ErrorException: Trying to access array offset on value of type null
in APPPATH\Controllers\ProcessController.php on line 37.
 1 APPPATH\Controllers\ProcessController.php(37): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\notlarimnet\\app\\Controllers\\ProcessController.php', 37, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:30:31 --> ErrorException: Trying to get property 'username' of non-object
in APPPATH\Controllers\ProcessController.php on line 37.
 1 APPPATH\Controllers\ProcessController.php(37): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get property \'username\' of non-object', 'C:\\xampp\\htdocs\\notlarimnet\\app\\Controllers\\ProcessController.php', 37, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:39:30 --> ErrorException: Undefined property: App\Controllers\ProcessController::$processModel
in APPPATH\Controllers\ProcessController.php on line 36.
 1 APPPATH\Controllers\ProcessController.php(36): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined property: App\\Controllers\\ProcessController::$processModel', 'C:\\xampp\\htdocs\\notlarimnet\\app\\Controllers\\ProcessController.php', 36, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:46:44 --> TypeError: Argument 1 passed to CodeIgniter\BaseModel::findAll() must be of the type int, array given, called in C:\xampp\htdocs\notlarimnet\app\Controllers\ProcessController.php on line 36
in SYSTEMPATH\BaseModel.php on line 621.
 1 APPPATH\Controllers\ProcessController.php(36): CodeIgniter\BaseModel->findAll([...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:48:46 --> TypeError: Argument 1 passed to CodeIgniter\BaseModel::findAll() must be of the type int, string given, called in C:\xampp\htdocs\notlarimnet\app\Controllers\ProcessController.php on line 36
in SYSTEMPATH\BaseModel.php on line 621.
 1 APPPATH\Controllers\ProcessController.php(36): CodeIgniter\BaseModel->findAll('a')
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:48:58 --> TypeError: Argument 1 passed to CodeIgniter\BaseModel::findAll() must be of the type int, string given, called in C:\xampp\htdocs\notlarimnet\app\Controllers\ProcessController.php on line 36
in SYSTEMPATH\BaseModel.php on line 621.
 1 APPPATH\Controllers\ProcessController.php(36): CodeIgniter\BaseModel->findAll('username', 'a')
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:51:09 --> Error: Call to a member function find() on null
in APPPATH\Controllers\ProcessController.php on line 36.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
CRITICAL - 2024-03-02 21:51:23 --> Error: Call to undefined method CodeIgniter\Database\MySQLi\Result::find()
in APPPATH\Controllers\ProcessController.php on line 36.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\ProcessController->insert()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProcessController))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\notlarimnet\\public\\index.php')
